std_msgs/Empty empty
---
geometry_msgs/Point point
std_msgs/Float32    degrees
